modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -153,7 +160,13 @@ const ColorPicker = ({
   };
 
   return (
-    <Popup width={250} padding={Popup.Padding.s} data-testid="canvas-color-picker" {...elemProps}>
+    <Popup
```

Comment:
Heads up @laurenraddatz and @mannycarrera4 

I think in the (near) future, we need to remove Popup from the ColorPicker. That should be something that the end user composes together not something we ship with this component.

I'm going to track this in  #836